### ISSUE TITLE
Accessibility issue: Elements must meet minimum color contrast ratio thresholds on /hpc_docs/

### ISSUE THREAD

**Issue by jrlagrone** [SouthernMethodistUniversity/hpc_docs#393]
## What
An accessibility scan flagged the element `<a href="#" title="                                                                            1 Event                                                                    " class="cal-mini-events-date" data-date="2026-02-23">` on https://southernmethodistuniversity.github.io/hpc_docs/ because elements must meet minimum color contrast ratio thresholds. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=playwright.

To fix this, ensure the contrast between foreground and background colors meets wcag 2 aa minimum contrast ratio thresholds.

Specifically:

Fix any of the following:
-   Element has insufficient color contrast of 2.16 (foreground color: #ffffff, background color: #ff972f, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

## Acceptance Criteria
- [ ] The specific axe violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific axe violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
